app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -0,0 +1,174 @@
+    });
+    console.log(newOptions);
+    return newOptions;
+  };
+  const getUsers = (input) => {
+    const url = formURL(input);
+    if (input) {
+      return fetch(url)
+        .then((response) => {
+          const arr = formOptions(response.content);
+          return ({ options: arr });
+        });
+    }
+    return Promise.resolve({ options: [] });
+  };
+  const menuRenderer = (params) => {
+    if (params.options[0].externalUser) {
+      return (
+        <div className={cx('Select-menu-outer')}>
+          <div className={cx('Select-menu')} role="listbox">
+            <div
+              onClick={() => params.selectValue(params.options[0])}
+              onMouseOver={() => params.focusOption(params.options[0])}
+            >
+              <div className={cx('invite-new-user')}>
+                <div className={cx('msg-icon')} />
+                <div className={cx('invite-info')}>
+                  <p className={cx('user-info')} >
+                    <FormattedMessage id={'InputUserSearch.inviteNewUser'} defaultMessage={'Invite {userEmail}'} values={{ userEmail: params.options[0].label }} />
+                  </p>
+                  <p className={cx('action-info')}>
+                    <FormattedMessage id={'InputUserSearch.inviteNewUserInfo'} defaultMessage={'Send invite via e-mail'} />
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    }
+    return (
+      <div className={cx('Select-menu-outer')}>
+        <ScrollWrapper autoHeight autoHeightMax={200}>
+          <div className={cx('Select-menu')} role="listbox">
+            {params.options.map(option => (<div
+              key={option.userLogin}
+              onClick={() => {
+                if (!option.disabled) {
+                  params.selectValue(option);
+                }
+              }}
+              onMouseOver={() => params.focusOption(option)}
+              disabled={option.disabled}
+            >
+              <UserItem
+                userName={option.userName}
+                userLogin={option.userLogin}
+                isAssigned={option.isAssigned}
+                userAvatar={option.userAvatar}
+              />
+            </div>))}
+          </div>
+        </ScrollWrapper>
+      </div>
+    );
+  };
+  const isValidNewOption = ({ label }) => validate.email(label);
+  const newOptionCreator = option => ({ externalUser: true, label: option.label });
+  const promptTextCreator = label => (label);
+  const filterOption = (option, filter) =>
+    option.userName.toLowerCase().indexOf(filter.toLowerCase()) >= 0 ||
+    option.userLogin.toLowerCase().indexOf(filter.toLowerCase()) >= 0 ||
+    (option.email && option.email.toLowerCase().indexOf(filter.toLowerCase()) >= 0);
+  return (<div>
+    <Select.AsyncCreatable
+      className={'select2-search-users'}
+      cache={false}
+      loadOptions={getUsers}
+      filterOption={filterOption}
```

Comment:
Do we need addition filtering since data is supposed to be filtered on the backend?

Reply:
Yes, it's custom filter which needed for correct work select library